src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Schema/XamlTypeInvoker.cs

```diff
@@ -241,6 +229,11 @@ public virtual MethodInfo GetAddMethod(XamlType contentType)
         /// </SecurityNote>
         public virtual MethodInfo GetEnumeratorMethod()
         {
+            if (IsUnknown)
```

Comment:
what's this change for?

Reply:
ahh i see, this is to fix a null reference exception. i think you could just add a null check.

Reply:
I actively chose not to do this to match the implementation of `GetAddMethod` which checks for IsUnknown before calling a function on `_xamlType`